File: demo/functions/doc-pipe/build-document/handler.rb
$LOAD_PATH.unshift(*Dir['./lib/**/lib'])

require 'json'
require 'builder/main'

def build_document(event:, context:)
  build_pdf()

  {
    statusCode: 200,
    body: {
      message: 'Go Serverless v1.0! Your function executed successfully!',
      input: event
    }.to_json
  }
end

File: demo/functions/doc-pipe/extract-data/handler.js
'use strict'

const scraper = require('./scraper')

module.exports.extractData = async (event) => {
  const d = await scraper.getSomething()
  console.log(d)

  return {
    statusCode: 200,
    body: JSON.stringify(
      {
        message: 'Go Serverless v1.0! Your function executed successfully!',
        input: event
      },
      null,
      2
    )
  }
}
